lib/os.js

```diff
@@ -121,6 +121,43 @@ function endianness() {
 }
 endianness[Symbol.toPrimitive] = () => kEndianness;
 
+function networkInterfaces() {
+  const interfaceAddresses = getInterfaceAddresses();
+
+  return Object.entries(interfaceAddresses).reduce((acc, [key, val]) => {
+    acc[key] = val.map((v) => {
+      const protocol = v.family.toLowerCase();
+
+      return Object.assign({}, v, {
```

Comment:
How big of an itch would that scratch? :D
Cuz we would be adding a loc when we don't have to and I usually go by lesser the loc, lesser the chances of bugs in that area. But yeah, if it is a stylistic thing in the node project, I am more than happy to move it into a `const` up top.